src/internal/connector/exchange/service_iterators.go

```diff
@@ -89,7 +93,7 @@ func filterContainersAndFillCollections(
 			}
 		}
 
-		added, removed, newDelta, err := getJobs(ctx, qp.ResourceOwner, cID, prevDelta)
+		added, removed, newDelta, err := getter.GetAddedAndRemovedItemIDs(ctx, qp.ResourceOwner, cID, prevDelta)
```

Comment:
There is a slight stutter here: `getter.Get...`. Refactor could be `getter.ItemIDsAddedAndRemoved()`

Reply:
If the stutter were a concern, in this case it'd be better to rename the interface than the func.  Due to golang's duck typing, there can be many interfaces that a given struct may fulfill; it's more sane to have each interface avoid stutters than to burden the source func with that worry.